ddl/table_split_test.go

```diff
@@ -64,6 +64,22 @@ func (s *testDDLTableSplitSuite) TestTableSplit(c *C) {
 	for _, def := range pi.Definitions {
 		checkRegionStartWithTableID(c, def.ID, store.(kvStore))
 	}
+
+	// test split region when create table like
+	tk.MustExec("create table t1(i int)")
+	tk.MustExec("create table t2 like t1")
+	t2, err := infoSchema.TableByName(model.NewCIStr("mysql"), model.NewCIStr("tidb"))
```

Comment:
Why is it `mysql.tidb`?

Reply:
This test is useless, I remove it and add the new test in the front of this page